store/bolt/user_store.go

```diff
@@ -0,0 +1,124 @@
+package bolt
+
+import (
+	"errors"
+
+	"github.com/boltdb/bolt"
+	"github.com/gopheracademy/congo"
+	"github.com/gopheracademy/congo/store/bolt/internal"
+)
+
+var (
+	// ErrUserNotFound is returned when mutating a user that doesn't exist.
+	ErrUserNotFound = errors.New("user not found")
+
+	// ErrEmailRequired is returned when creating a user without an email.
+	ErrEmailRequired = errors.New("email address required")
+)
```

Comment:
Now that I'm looking at the code again, these errors should probably be moved to the `store` package so they can be shared by other store implementations.